src/EnergyPlus/MixedAir.cc

```diff
@@ -4568,10 +4572,18 @@ namespace MixedAir {
                     // simulate OA System if equipment exists other than the mixer (e.g., heating/cooling coil, HX, ect.)
 
                     // 1 - check min OA flow result
-                    Node(this->OANode).MassFlowRate = max(this->ExhMassFlow, OutAirMinFrac * Node(this->MixNode).MassFlowRate);
-                    Node(this->RelNode).MassFlowRate = max(Node(this->OANode).MassFlowRate - this->ExhMassFlow, 0.0);
-                    // save actual OA flow frac for use as min value for RegulaFalsi call
-                    minOAFrac = max(OutAirMinFrac, Node(this->OANode).MassFlowRate / this->MixMassFlow);
+                    if (this->FixedMin) {
+                        Node(this->OANode).MassFlowRate =
+                            min(max(this->ExhMassFlow, OutAirMinFrac * AirLoopFlow(AirLoopNum).DesSupply), Node(this->MixNode).MassFlowRate);
+                        Node(this->RelNode).MassFlowRate = max(Node(this->OANode).MassFlowRate - this->ExhMassFlow, 0.0);
+                        // save actual OA flow frac for use as min value for RegulaFalsi call
+                        minOAFrac = max(OutAirMinFrac, Node(this->OANode).MassFlowRate / AirLoopFlow(AirLoopNum).DesSupply);
```

Comment:
Why is this dividing by DesSupply? If we're shooting for the actual OAFrac here, I understand the line above using DesSupply to apply the fixedmin flow, but then the actual OAfrac will be higher if MixMassFlow < DesSupply.

Reply:
OutAirMinFrac is defined based on "Design Supply" flow as a basis. Can we compare it with a fraction calculated based on a different basis? However, this min OA faction is used as lower bound in the rootsolver and the rootsolver in MultiCompControlTempResidual function uses MixMassFlow. I think it makes sense to change to:

minOAFrac = max(OutAirMinFrac, Node(this->OANode).MassFlowRate / (this-> MixMassFlow);